packages/vm/src/calls.rs

```diff
@@ -677,6 +680,45 @@ where
     )
 }
 
+#[cfg(feature = "ibc2")]
+pub fn call_ibc2_acknowledge_receive<A, S, Q, U>(
+    instance: &mut Instance<A, S, Q>,
+    env: &Env,
+    msg: &Ibc2PacketReceiveMsg,
+) -> VmResult<ContractResult<IbcReceiveResponse<U>>>
+where
+    A: BackendApi + 'static,
+    S: Storage + 'static,
+    Q: Querier + 'static,
+    U: DeserializeOwned + CustomMsg,
+{
+    let env = to_vec(env)?;
+    let msg = to_vec(msg)?;
+    let data = call_ibc2_acknowledge_receive_raw(instance, &env, &msg)?;
+    let result = from_slice(&data, deserialization_limits::RESULT_IBC_PACKET_RECEIVE)?;
+    Ok(result)
+}
+
+#[cfg(feature = "ibc2")]
+pub fn call_ibc2_acknowledge_receive_raw<A, S, Q>(
+    instance: &mut Instance<A, S, Q>,
+    env: &[u8],
+    msg: &[u8],
+) -> VmResult<Vec<u8>>
+where
+    A: BackendApi + 'static,
+    S: Storage + 'static,
+    Q: Querier + 'static,
+{
+    instance.set_storage_readonly(false);
+    call_raw(
+        instance,
+        "ibc2_acknowledge_receive",
```

Comment:
I feel like `ibc2_packet_acknowledgement` or `ibc2_packet_ack` would fit better with the naming we used for the other entrypoints (like `ibc2_packet_timeout`, `ibc2_packet_receive`, etc.). Or was this discussed before?

Reply:
No, wasn't discussed before, I just kept the name from the initial design doc. But agreed, packet_ack is clearer